getting_started.Rmd

```diff
@@ -8,17 +8,30 @@ output:
 
 ## Prerequisites for using OTTR
 
-- Please take a look at the [code of conduct](https://github.com/jhudsl/OTTR_Template/blob/main/code_of_conduct.md).
+First please take a look at our [code of conduct](https://github.com/jhudsl/OTTR_Template/blob/main/code_of_conduct.md) if you wish to contribute to OTTR. 
+
+OTTR relies on R Markdown and GitHub actions, however, you do not need to be familiar with either R Markdown or GitHub to use OTTR!
```

Comment:
```suggestion
OTTR relies on R and GitHub actions, however, you do not need to be familiar with either R or GitHub actions to use OTTR! (However, you should be familiar with markdown)
```

Reply:
Just to clarify, because in a quick read this seems like its saying you don't need to know markdown or github but then we say you do need to know markdown and github.

Reply:
> OTTR relies on R and GitHub actions, however, you do not need to be familiar with either R or GitHub actions to use OTTR! (However, you should be familiar with markdown)

what about this: 

OTTR relies on R Markdown and GitHub actions. You do not need to be familiar with either R Markdown or GitHub to get started using OTTR, as we will guide you through the process! However, we recommend reading up a bit about both to better understand how they work in general. These readings only take a few minutes.